pkg/volume/glusterfs/glusterfs.go

```diff
@@ -671,6 +680,10 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 		return nil, fmt.Errorf("StorageClass for provisioner %s must contain 'resturl' parameter", glusterfsPluginName)
 	}
 
+	if cfg.gid == 0 {
+		cfg.gid = defaultGid
+	}
```

Comment:
With the defaultGid being 0, you could actually skip this hunk and the defaultGid altogether withouth changing the behavior of the patch. Only the default of 0 would be implicit.